src/services/telegram.ts

```diff
@@ -0,0 +1,51 @@
+import axios from 'axios';
+import _pipe from 'lodash/fp/pipe';
+import _map from 'lodash/fp/map';
+import _get from 'lodash/fp/get';
+import _uniqBy from 'lodash/fp/uniqBy';
+import _filter from 'lodash/fp/filter';
+import { env } from '../config';
+import mapKeys from '../utils/mapKeys';
+import { KEY_MAPPERS } from '../constants';
+import { service as telegramService } from '../components/telegram-subscriptions';
+
+const getMessage = _get('message');
```

Comment:
minor: this method looks small and simple enough so you can just use `_get('message')` inline in `getSubscribers`